docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,178 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter's security-related features start with support for cryptographic
+signing with public/private keys, then continue with additional levels of
+security and customizable permissions. This topic summarizes how Splinter uses
+key-based signing, provides Biome to map user credentials to keys, and supports
+organization-based smart permissions with the Pike smart contract. It also
+describes Splinter's port requirements and security recommendations for REST API
+connections.
+
+## Cryptographic Signing with Public/Private Keys
+
+Like [Hyperledger Sawtooth](https://sawtooth.hyperledger.org/) and other
+distributed ledgers, Splinter supports [key-based cryptographic
+signing](https://en.wikipedia.org/wiki/Public-key_cryptography).
+
+Scabbard, the default distributed ledger service included in Splinter, uses
+public/private keys to sign all transactions that affect shared state. Each user
+and transaction-submitting entity has a public key and a private key for signing
+transactions. This includes smart contract operations (uploads, configuration,
+and changes), as well as transaction-related functions performed by the Splinter
+daemon and application-specific services.
+
+Splinter also uses public/private keys to sign non-transaction operations such
```

Comment:
Removing the word "non-transaction" here would be fine, because "to sign operations such as" provides the intent which is to differentiate it from ledger transactions.

Reply:
Fixed.  It now says "Splinter also uses public/private keys to sign operations such as ...".